src/utils/__test__/index.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import { createLocalStorage, debounce } from "utils";
+
+import "../axios";
+
+describe("utils/debounce:", () => {
+  it("call multi times (2)", async () => {
+    const fn = jest.fn();
+    const debounced = debounce(fn, 100);
+    debounced();
+    debounced();
+    debounced();
+    await Promise.delay(100);
+
+    expect(fn).toBeCalledTimes(2);
+  });
+
+  it("call multi times (3)", async () => {
+    const fn = jest.fn();
+    const debounced = debounce(fn, 100);
+    debounced();
+    debounced();
+    debounced();
+    await Promise.delay(100);
+    debounced();
+    debounced();
+    debounced();
+    await Promise.delay(100);
+
+    expect(fn).toBeCalledTimes(3);
+  });
+
+  it("call with default delay", async () => {
+    const fn = jest.fn();
+    const debounced = debounce(fn);
+    debounced();
+    debounced();
+    debounced();
+    await Promise.delay(100);
+    debounced();
+    debounced();
+    debounced();
+    await Promise.delay(100);
+
+    expect(fn).toBeCalledTimes(3);
+  });
+});
+
+describe("utils/localStorage:", () => {
+  it("initial default value", () => {
+    const storage = createLocalStorage("Storage-test", false);
+
+    expect(storage.get()).toBe(false);
+  });
+
+  it("should change value", () => {
+    const storage = createLocalStorage("Storage-test", false);
+    storage.set(true);
+    expect(storage.get()).toBe(true);
+  });
+});
+
+describe("utils/Promise.delay:", () => {
+  it("initial default value", async () => {
+    const now = Date.now();
+    await Promise.delay();
+    expect(Date.now() - now >= 0).toBe(true);
```

Comment:
Actually I think this expect statement cannot tell anything, because the `delay` is a promise, you await it mean it will be pushed to the queue, thus of course `Date.now() - now` is `true` anyway

Reply:
I was update this case for this in another PR.
Thank for comment.